README.md

```diff
@@ -1,71 +1,75 @@
 
 There is a `STEERING_LKA`-ish message and more in some new Toyotas that currently has an "authentication code" scheme appended to the end. The algorithm and security system for this "authentication code" is somewhat known for certain vehicles but requires a key that is unique to each vehicle to be extracted or smuggled out of the vehicle (https://icanhack.nl/blog/secoc-key-extraction/). Not all vehicles are able to have their keys extracted with what is currently known. Without the key or knowledge of the system, third parties like comma and users cannot control the vehicle. While vehicles that have had their keys smuggled out are currently working with openpilot.
 
-## List of vehicles with Toyota ECU Security Key
+---
+
+## Cars
+
+### Successfully running openpilot
+
+These cars can run openpilot but are not listed on https://comma.ai/vehicles or [CARS.md](https://github.com/commaai/openpilot/blob/master/docs/CARS.md) because Comma (the company) understandably doesn't want to own the security key hacking process. Follow the [Setup Guide](#setup-guide) below and you'll have it working.
 
-Known vehicles with Toyota ECU Security Key on the Forward Recognition Camera of which OP **doesn't** currently support right now:
+* 2021-2023 RAV4 Prime
+* 2021-2023 Sienna
+* 2020+ Yaris Hybrid (EUDM/JDM/MXDM)
+  * Needs extra steps. See [Support Status Overview](#support-status-overview).
 
-> [!NOTE]
-> A notable vehicle not included here is the US-made (VIN does not start with `J`) 2023 Corolla. It does not appear to have ECU Security Key or SecOC steps when replacing the forward camera. If you have one of these US Corollas, stop by the [comma Discord](https://discord.comma.ai). It's exact standing or nature is unknown.
+### May be possible to hack but hasn't been tried
 
-> [!NOTE]
-> Bold vehicles are vehicles we suspect might work with the current key dumping procedures but no one has stopped by and/or attempted on them yet. It is speculation.
+If you have one of these cars, please stop by the [comma Discord](https://discord.comma.ai)'s #toyota-security channel - we need more information from people like you.
 
-> [!NOTE]
-> Vehicles that are crossed out are supported by openpilot but are not on https://comma.ai/vehicles because they are not fully officially supported. The primary reason stems from the key extraction process being produced from a non-comma.ai process. That said, if you have the key, it will work.
+* 2023 US-made Corolla (VIN does not start with `J`)
+  * Uses TSS 3.0 but does not appear to have ECU Security Key or SecOC steps when replacing the forward camera. It's unknown whether it has TSK, and if yes in what form. Maybe they just don't do the pairing thing but hardcode a key. No one knows.
+* 2023 TMC/JP-made Corolla[^3]
+  * May be similar to US-made? Unknown.
```

Comment:
I think it would be best to put this back down to `Not hacked and can't run openpilot` for now. It's more similar to the others than the interesting ones in this category.